app/src/main/res/values/untraslatable.xml

```diff
@@ -133,6 +135,10 @@
 		<item quantity="one">@string/TestResults_Summary_Websites_Hero_Reachable_Singular</item>
 		<item quantity="other">@string/TestResults_Summary_Websites_Hero_Reachable_Plural</item>
 	</plurals>
+	<plurals name="TestResults_Overview_Circumvention_RiseupVPN_Blocked" translatable="false">
+		<item quantity="one">@string/TestResults_Overview_Circumvention_RiseupVPN_Blocked_Singular</item>
+		<item quantity="other">@string/TestResults_Overview_Circumvention_RiseupVPN_Blocked_Plural</item>
```

Comment:
@cyBerta we still use the plurals here.
If you want to help us improve our translation system, let me know we can discuss it in a github issue.
Very appreciated.

Reply:
Sure, if it's not super urgent to be finished, I can definitely work on that. +1 for leaving this issue aside here and for opening an extra ticket for it.

Reply:
I'm gonna move ahead, merge this issue and opened another ticket about the plurals: https://github.com/ooni/translations/issues/16
I still don't understand why our approach (string.xml for strings and untransatable.xml for plurals) it can cause that problem as we still use the quantities.